spec/requests/settings_spec.rb

```diff
@@ -3,6 +3,25 @@
 #
 describe "Settings API" do
   let(:api_settings) { Api::ApiConfig.collections[:settings][:categories] }
+  let(:server) { FactoryGirl.create(:miq_server) }
+  let(:super_admin) { FactoryGirl.create(:user, :role => 'super_administrator', :userid => 'alice', :password => 'alicepassword') }
+
+  context "Settings Update" do
+    it "updates to settings return a BadRequestError upon failed validation" do
+      api_basic_authorize(:user => super_admin.userid, :password => super_admin.password)
+
+      patch(api_server_settings_url(nil, server), :params => {:authentication => {:mode => "bogus_auth_mode"}})
```

Comment:
The other server settings specs are in `spec/requests/servers_spec.rb` - would it make sense for this to live there too?

Reply:
starting going that way, but then realizing I'm really testing a settings update handling that's generic to all and it didn't make sense to duplicate for servers, zone, region, so I put that here.

Reply:
ahh good point, I'll move it the proper file.